VAMobile/src/screens/HealthScreen/Appointments/AppointmentDetailsCommon/AppointmentReason.tsx

```diff
@@ -14,20 +14,24 @@ type AppointmentReasonProps = {
 const AppointmentReason: FC<AppointmentReasonProps> = ({ attributes }) => {
   const { t } = useTranslation(NAMESPACE.COMMON)
   const theme = useTheme()
-  const { reason, phoneOnly } = attributes || ({} as AppointmentAttributes)
+  const { reason, phoneOnly, appointmentType, serviceCategoryName } = attributes || ({} as AppointmentAttributes)
   const isPendingAppointment = isAPendingAppointment(attributes)
 
   if (!phoneOnly && (isPendingAppointment || !reason)) {
     return <></>
   } else {
-    const phoneApptReason = reason || t('notNoted')
+    const apptReason = reason || t('notNoted')
     return (
       <Box mt={phoneOnly ? undefined : theme.dimensions.standardMarginBetween}>
         <TextView variant="MobileBodyBold" accessibilityRole="header" mb={theme.dimensions.condensedMarginBetween}>
-          {phoneOnly ? t('upcomingAppointmentDetails.sharedProvider') : t('upcomingAppointmentDetails.reason')}
+          {phoneOnly || (appointmentType === AppointmentTypeConstants.VA && serviceCategoryName !== 'COMPENSATION & PENSION')
```

Comment:
I noticed this logic is also used in many files also. Maybe it'd be more worth it to put this in a util function instead of making `'COMPENSATION & PENSION'` a const. This could improve readability and reduce redundancy

Reply:
true, however the next set of appt work I will be able to condense a ton of things so I feel it is fine leaving it like this